notebook/static/notebook/js/actions.js

```diff
@@ -462,38 +462,23 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
-        'show-all-line-numbers': {
```

Comment:
Actions are quite public API - they're used to define keyboard shortcuts - so I'd rather not remove them without a good reason.